cherrypy/lib/httputil.py

```diff
@@ -66,6 +68,29 @@ def protocol_from_http(protocol_str):
     return int(protocol_str[5]), int(protocol_str[7])
 
 
+def passes_if_range_check(if_range_header):
```

Comment:
I think the naming of this function could be improved.

Reply:
What would you suggest?

Reply:
maybe 
```suggestion
def matches_if_range_condition(if_range_header):
```

Reply:
I like that. Adding that into the changes.